src/solvers/prop/prop_conv_solver.h

```diff
@@ -124,10 +127,24 @@ class prop_conv_solvert : public conflict_providert,
 
   virtual void ignoring(const exprt &expr);
 
-  // deliberately protected now to protect lower-level API
+  /// Actually adds the constraints to `prop`
```

Comment:
Explicitly mention that `set_to` is a wrapper around this, and which one ought to be used

Reply:
made private

Reply:
`do_set_to` is not a great name, why not naming it `add_constraint_to_prop`?